### ISSUE TITLE
Why is my my predicted regression value too small?

### ISSUE THREAD

**Issue by lxxxsd** [timeseriesAI/tsai#498]
predictions (for regression)
![image](https://user-images.githubusercontent.com/96508261/166131181-c2f7ffa0-689f-4648-8eb2-e49d0f77bbe0.png)
The absolute value of my target value is greater than 0.1, but my predicted regression value is less than 0.1.
![image](https://user-images.githubusercontent.com/96508261/166131241-f39daa8e-24fc-4f7d-be47-4d3767c7a265.png)

tfms  = [ToFloat(), ToFloat()] # convert int to float
batch_tfms = TSStandardize()
dls = get_ts_dls(X, y, splits=splits, tfms=tfms, batch_tfms=batch_tfms, bs=2048)
model = TST(dls.vars, dls.c, dls.len, dropout=0.3, fc_dropout=0.9)
learn = ts_learner(dls, model, loss_func=MSELossFlat(), metrics=[mae, rmse], cbs=ShowGraph())
# learn.lr_find()
learn.fit_one_cycle(2, 1e-4)

PATH = Path('./models/ShaoHuaChuXin.shcx')
PATH.parent.mkdir(parents=True, exist_ok=True)
learn.export(PATH)

**Comment by oguiza** [comment]
There are many factors that play a role in how good a model is. It's impossible to say what could be happening without knowing your dataset. 
How is your training set doing? Have you reached a good performance there? That's always the first step.
A few things I can spot: 
* fc_dropout seems very high (.9)
* dropout seems also high (.3)
* the number of epochs may be too low (2).

**Comment by lxxxsd** [comment]
thank you!
